app/static/i18n/de.json

```
@@ -174,13 +174,13 @@
         "STACK_OVERFLOW_TITLE": "Stack Overflow",
         "STACK_OVERFLOW_MSG": "Frage auf Stack Overflow.",
         "DISCOURSE_TITLE": "Discourse",
-        "DISCOURSE_MSG": "Tausch dich mit CrateDB Ingenieuren und anderen Benutzern in unserer Discourse Community aus.",
+        "DISCOURSE_MSG": "Tausche Dich mit CrateDB Nutzern und Entwicklern aus, um Antworten zu Deinen Fragen zu erhalten.",
```

```suggestion
        "DISCOURSE_MSG": "Tausche Dich mit Nutzern und Entwicklern von CrateDB aus, um Antworten zu Deinen Fragen zu erhalten.",
```
Otherwise, there should be a hypen, like "mit CrateDB-Nutzern und -Entwicklern", which maybe looks a bit odd.

---

Sounds good, thanks @hammerhead